popup.js

```diff
@@ -1,19 +1,27 @@
-const getPackageName = url => url.split('/').filter(Boolean).slice(0, 4).slice(-1);
+const getPackageName = url => {
+    const parts = url.split('/').filter(Boolean);
+    const [org, name] = parts[1].endsWith('npmjs.com')
+        ? parts.slice(3, 5)
+        : parts.slice(2, 4);
+    return org.startsWith('@')
+        ? `${org}/${name}`
+        : name || org;
+};
 const getBundlephobiaUrl = name => `https://bundlephobia.com/result?p=${name}&src=bundlephobia-extension`;
 const getIframe = () => document.querySelector('iframe');
 
 chrome.tabs.query({
     active: true,
     currentWindow: true
-  }, ([currentTab]) => {
```

Comment:
Just a little nit a out these changes, would you mind not modifying the indentation?

Reply:
Sure, just fixed the indentation.